OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/createOIMDomain.py

```diff
@@ -291,16 +291,16 @@ def extendOimDomain(self, domainHome, db, dbPrefix, dbPassword, user, password,
 
         cd('/')
         cd('/Server/AdminServer')
-        cmo.setListenAddress('oimadmin')
+        #cmo.setListenAddress('oimadmin')
```

Comment:
Please do not just comment out lines. If you don't need the line, remove it. This should apply to all commented lines.